sci-libs/vtk/files/vtk-9.0.3-IO-FFMPEG-support-FFmpeg-5.0-API-changes.patch

```diff
@@ -0,0 +1,141 @@
+From bad8f3e9d0aad03e8f2aff9524cb3c5f2d10ccaf Mon Sep 17 00:00:00 2001
```

Comment:
Link to where it came from at top of patch + any relevant Gentoo bugs please

Reply:
I hope I catched any related bug.